lib/chf/data_fixes/work_dates.rb

```diff
@@ -0,0 +1,26 @@
+module CHF
+  module DataFixes
+    class WorkDates
+      attr_reader :work
+
+      def initialize(work)
+        @work = work
+      end
+
+      # possibly makes changes to work (without saving), returns true
+      # if changes were made, so caller can do expensive save.
+      def change
+        updated = false
```

Comment:
Yeah there is a separate `visibility_changed?` as well. So maybe that is the only one that isn't caught? idk; I agree it seems better not to rely on it.